src/components/ThemeSwitcher.jsx

```diff
@@ -0,0 +1,36 @@
+import Label from './Label';
+import Input from './Input';
+
+const ThemeSwitcher = ({ value, htmlFor }) => {
+  return (
+    <div class="label-input ml-2 flex h-12 w-1/5 flex-col items-center">
+      <form class="theme-form" name="themeForm">
+        <div class="labels flex h-6 w-16 items-center justify-between">
+          <Label value="1" for="theme-choice-one" htmlFor="theme-choice-one" />
+          <Label value="2" for="theme-choice-two" htmlFor="theme-choice-two" />
+          <Label value="3" for="theme-choice-three" htmlFor="theme-choice-three" />
```

Comment:
Let's refactor this to use `Array.map`, this we we keep the code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and reusable.

We can create an array of object that can be used to map these component and the Input component since they have some values in common.